src/lib/configs.ts

```diff
@@ -50,6 +50,26 @@ async function loadConfigsFromFile() {
   }
 }
 
+export function findSpecFile(baseDir): string{
+  const files = ['asyncapi.yaml', 'asyncapi.json', 'asyncapi.yml']
+  let indexOfFoundFile = -1
+
+  for (let index = 0; index < files.length; index++) {
+    const file = files[index]
+    if(existsSync(path.resolve(baseDir, file))){
+      if(indexOfFoundFile === -1){
+      indexOfFoundFile = index
+      } else {
+        throw new Error('Multiple AsyncAPI files found at the root of your project. Please make sure to choose one in your config file.')
+      }
+    }
+  }
+  
+  if(indexOfFoundFile === -1){
+    throw new Error('AsyncAPI file was not found at the root of your project. you can set its path in config file.')
```

Comment:
Would be great to provide help on how "you can set its path in config file". Have a look at https://wix-ux.com/when-life-gives-you-lemons-write-better-error-messages-46c5223e1a2f.

Also, when this error throws, it will be showing the stack trace and all the stuff that comes with throwing an error. This information is not relevant as it's not a failure in the code. Please consider "catching" the error in a function above and simply show a beautiful and informative error message using https://github.com/asyncapi/glee/blob/master/src/lib/logger.ts#L123.